src/SFA.DAS.EmployerIncentives.Commands/Persistence/ApprenticeshipIncentiveDomainRepository.cs

```diff
@@ -110,6 +110,13 @@ public async Task<Domain.ApprenticeshipIncentives.ApprenticeshipIncentive> FindB
             return _apprenticeshipIncentiveFactory.GetExisting(incentive.Id, incentive);
         }
 
+        public async Task<List<Domain.ApprenticeshipIncentives.ApprenticeshipIncentive>> FindByAccountLegalEntity(long accountLegalEntityId)
```

Comment:
Does this need all the preceding namespace declarations? Doesn't this normally happen when two namespaces are the same as the class name?

Reply:
It's because ApprenticeshipIncentive is also defined as a namespace - SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive as introduced by this project - so the namespace prefix has to be explicitly defined